src/pages/dashboardUsers/components/UsersTableProduct/components/UserProductTableColumns.tsx

```diff
@@ -53,7 +53,7 @@ export function buildColumnDefs(
     {
       field: 'userRole',
       cellClassName: 'justifyContentBold',
-      headerName: 'RUOLI',
+      headerName: 'Ruolo 🡣',
```

Comment:
The arrow inside the header cell of a table represents the direction of the sort, it's not part of the label
Moreover, ASIS neither column should be sortable inside this table
```suggestion
      headerName: 'Ruolo',
```